cli/azd/pkg/devcenter/manager.go

```diff
@@ -15,6 +17,9 @@ import (
 	"golang.org/x/exp/slices"
 )
 
+// ADE Bicep deployments have a name of a date like string followed by a number
+var bicepDeploymentNameRegex = regexp.MustCompile(`^\d{4}-\d{2}-\d{2}-\d+$`)
```

Comment:
Are we guaranteed on stability of the naming contract `YYYY-MM-DD-<digit>`? Just hoping that the service wouldn't shift the naming convention and end up breaking the integration,

Reply:
We have asked them to apply the same tagging strategy that exists for their ARM runner deployments.  This is just a stop gap solution for Bicep until that happens and before we have the capacity to integrate with their outputs API.